src/System.CommandLine.DragonFruit.Tests/CommandLineTests.cs

```diff
@@ -46,11 +47,11 @@ public void It_executes_method_synchronously_with_string_option()
         }
 
         [Fact]
-        public async Task It_shows_help_text_based_on_XML_documentation_comments()
+        public async Task It_shows_help_text_based_on_XML_documentation_comments_without_para()
```

Comment:
Test naming nitpick. 🤓 I would keep this test name the same, since the scenario is unchanged. The new test is adding detail to that scenario so something like `When_XML_documentation_comment_contains_a_para_tag_then_help_is_written_with_a_newline` would be good to bring attention to the expected behavior.

Reply:
Done!